lms/envs/test.py

```diff
@@ -579,7 +579,7 @@
 NODEBB_ENDPOINT = "http://local.philanthropyu.org:4567"
 # replace NODEBB_MASTER_TOKEN with value from your setup
 NODEBB_MASTER_TOKEN = 'test-master-token-nodebb'
-MANDRILL_API_KEY = 'test_mandrill_api_key_part1-part2-part3'
+MANDRILL_API_KEY = os.environ.get('MANDRILL_API_KEY', 'test_mandrill_api_key_part1-part2-part3')
```

Comment:
As due to invalid mandril api key, related unit tests were failing. In this way it comes in the scope of this ticket.

Reply:
I still feel that it should be a part of Ahmed's PR, related to CI/CD. That way, it'll be easier to track all the changes that were done in test files as part of implementing the CI/CD pipeline. Let Ahmed share his thoughts on the matter, before we proceed.

Another reason for my suggested approach is that there would be other test cases as well, which would be failing due to invalid mandril api key. The unit tests related to this PR are probably not an isolated case that are failing because of the mentioned issue.

Reply:
I guess we can do it both ways, if @zeshanarifios you want to get it merged under this ticket then I'd request to make a separate commit for this one change (commit message accordingly), it would be easier to track. OR you can revert this I am creating follow up PR, have made some fixes, I'll push this one as well.

Reply:
I'll insist on the 2nd approach, i.e. Zeshan reverts it, and Ahmed you incorporate it in the follow-up PR. Let's keep all related changes in CI/CD ticket.

Reply:
Make sense. Sure I'll tackle it in my PR.